qutip/qobj.py

```diff
@@ -502,11 +502,12 @@ def __mul__(self, other):
                     # to have uneven length (non-square Qobjs).
                     # We use None as padding so that it doesn't match anything,
                     # and will never cause a partial trace on the other side.
-                    mask = [l == r == 1 for l, r in zip_longest(dims[0], dims[1],
```

Comment:
mask = [l == r == 1 for l, r in zip_longest(dims[0], dims[1])]

Reply:
could you explain your suggestion? thanks!